platform-dom/src/main/java/com/softicar/platform/dom/elements/input/auto/DomAutoCompleteIdMapFactory.java

```diff
@@ -0,0 +1,51 @@
+package com.softicar.platform.dom.elements.input.auto;
+
+import com.softicar.platform.common.core.entity.EntityIdFromDisplayStringExtractor;
+import java.util.Map;
+import java.util.Optional;
+import java.util.TreeMap;
+
+/**
+ * Attempts to extract numeric identifiers (IDs) from the displayed strings of
```

Comment:
`Attempts to extract` -> `Extracts` ... let's be more optimistic in the introductory sentence.

Reply:
ok